cmd/bosun/sched/check.go

```diff
@@ -185,6 +186,44 @@ func (s *Schedule) RunHistory(r *RunHistory) {
 	if checkNotify && s.nc != nil {
 		s.nc <- true
 	}
+	// Instrument alert states.
+	severityCount := make(map[string]int64)
+	lastAbnormalCount := make(map[string]int64)
+	unAckedCount := map[bool]int64{true: 0, false: 0}
+	activeCount := map[bool]int64{true: 0, false: 0}
```

Comment:
Using a map for these seems a bit confusing. Why not just have two variables activeCount/inactiveCount? / ackedCount / unAckedCount?